websites/K/KissAnime/presence.ts

```diff
@@ -0,0 +1,89 @@
+const presence = new Presence({
+    clientId: "877373455941664799"
+  }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused",
+    browsing: "presence.activity.browsing"
+  });
+let tv: boolean,
+  video = {
+    duration: 0,
+    currentTime: 0,
+    paused: true
+  };
+
+/**
+ * Get Timestamps
+ * @param {Number} videoTime Current video time seconds
+ * @param {Number} videoDuration Video duration seconds
+ */
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  const startTime = Date.now(),
+    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
+
+presence.on(
+  "iFrameData",
+  (data: { duration: number; currentTime: number; paused: boolean }) => {
+    video = data;
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const data: PresenceData = {
+    largeImageKey: "KissAnime"
+  };
+
+  if (
+    video !== null &&
+    !isNaN(video.duration) &&
+    document.location.pathname.includes("/watch")
+  ) {
+    tv =
+      document.querySelector("#episodes .episodes a.active") !== null &&
+      /\d/.test(
+        document.querySelector("#episodes .episodes a.active").textContent
+      )
+        ? true
+        : false;
+
+    const timestamps = getTimestamps(
+      Math.floor(video.currentTime),
+      Math.floor(video.duration)
+    );
```

Comment:
You can't use `getTimestamps()` anymore, you need to use:`presence.getTimestamps()` to prevent useless additional code